homeassistant/components/lektrico/entity.py

```diff
@@ -0,0 +1,35 @@
+"""Entity classes for the Lektrico integration."""
+
+from __future__ import annotations
+
+from lektricowifi import InfoForCharger
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import LektricoDeviceDataUpdateCoordinator
+from .const import DOMAIN
+
+
+class LektricoEntity(CoordinatorEntity[LektricoDeviceDataUpdateCoordinator]):
+    """Define an Lektrico entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: LektricoDeviceDataUpdateCoordinator,
+        friendly_name: str,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        _infoForCharger: InfoForCharger = coordinator.data
```

Comment:
Why the underscore and please use snake_case

Reply:
I made the changes, thanks.